data/files/00c1c3552fd04e8db4cf6252df7a60d9/pitzer_14.csv

```diff
@@ -0,0 +1,193 @@
+"T [C]","P [bar]",0.1,0.25,0.5,0.75,1,2,3,4,5,6
```

Comment:
It looks like part of the header got cut-off here - the numbers (0.1, 0.25, etc.) all represent a composition (if I recall correctly), but that is not contained in the header here.